metaworld/envs/mujoco/sawyer_xyz/sawyer_bin_picking.py

```diff
@@ -78,13 +79,6 @@ def _set_goal_xyz(self, goal):
         qvel = self.data.qvel.flat.copy()
         self.set_state(qpos, qvel)
 
-    def _set_obj_xyz(self, pos):
```

Comment:
so where does the setting of obj_xyz happen now?

Reply:
There has always been a `_set_obj_xyz` function in the superclass, a bunch of envs have just been overriding it for no reason. So now they'll just call the superclass function.